contracts/staking/test/unit_tests/finalizer_test.ts

```diff
@@ -142,16 +135,16 @@ blockchainTests.resets('Finalizer unit tests', env => {
 
     async function assertFinalizationLogsAndBalancesAsync(
         rewardsAvailable: Numberish,
-        activePools: ActivePoolOpts[],
+        numPoolsToFinalize: ActivePoolOpts[],
```

Comment:
This parameter name doesn't make sense?

Reply:
It does not. I suppose this code was too grepable 😄